src/main/java/seedu/duke/commands/WithdrawCommand.java

```diff
@@ -0,0 +1,77 @@
+package seedu.duke.commands;
+
+import seedu.duke.AccountList;
+import seedu.duke.Currency;
+import seedu.duke.constants.ErrorMessage;
+import seedu.duke.constants.Message;
+import seedu.duke.exceptions.InsufficientAccountBalance;
+import seedu.duke.exceptions.InvalidAddCommandException;
+import seedu.duke.exceptions.NegativeWithdrawalAmountException;
+import seedu.duke.exceptions.NoAccountException;
+import seedu.duke.ui.Ui;
+
+/**
+ * This class is used to deal with the withdrawCommand.
+ */
+public class WithdrawCommand extends Command {
+    private Currency currency;
+    private int amount;
+
+    /**
+     * @param input   The user input including the command.
+     */
+    public WithdrawCommand(String input) {
+        super(false, input);
+    }
+
+    private Currency getCurrency(String currencyString) {
+        return Currency.valueOf(currencyString);
+    }
+
+    private void processCommand() throws InvalidAddCommandException, NegativeWithdrawalAmountException {
```

Comment:
Perhaps you would like to rename the `InvalidAddCommandException` as this has to deal with the Withdrawal instead of adding?

Reply:
yeap, my bad